IndomitableIbises/Bangazon-Sprint-2
# Department Model/List View completed.  Added urls.py

## Link to Ticket
References #5 

## Description of Proposed Changes

- Added urls.py to bang folder
- Added `'bang.apps.BangConfig'` to app/settings.py
- Added `path('bang/', include('bang.urls'))` to app/urls.py AND imported **include**
- Added to bang/urls.py:
```
from django.urls import path
from . import views


app_name = 'bang'
urlpatterns = [

]
```
- Created **Department Model**
- Created **Department View** with **DepartmentListView**

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout RC-Department
```
- Make migrations to create the Department Table `python manage.py makemigrations` and check to see if it's created in DBBrowser


## Impacted Areas in Application

List general components of the application that this PR will affect:

* app/settings.py
* app/urls.py
* bang/models/department.py
* bang/models/department_view.py
* bang/urls.py

## Mentions @username

Tag users that need to review this code
@Levi-Schubert @sirwin4 
## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
